app/matchers.js

```diff
@@ -3,7 +3,9 @@ export function transferMatcher(text) {
 }
 
 export function closeMatcher(text) {
-  return text.match(/(?:^| | \r\n|\n)\/close (not-planned)|\/close/);
+  return text.match(
+    /(?:^| | \r\n|\n)\/close (not-planned)|(?:^| | \r\n|\n)\/close/
```

Comment:
the bug was introduced because it was trying not to match `/closenot-planned` which in the naïve regex of:
```
/(?:^| | \r\n|\n)\/close ?(not-planned)?/
```

would match, there's probably a simpler fix than this but I'm not aware of it.

I checked all other matchers and this is the only one with the issue